content/updates/2024-08-14-call-for-sponsors/index.mdx

```diff
@@ -3,6 +3,7 @@ title: Call for Sponsors
 description: Help us make Satsummit Lisbon a success by sponsoring the conference.
 editions:
   - edition: 2024-lisbon
+    featured: 1
```

Comment:
To set an update as featured just add a numeric weight value to the edition you want